src/app/app.component.ts

```diff
@@ -20,14 +20,24 @@ export class AppComponent implements OnInit  {
     private searchSearch: SearchService,
     private elRef: ElementRef,
     private uiActionsService: UiActionsService,
-  ) {}
+    ) {
+      this.router.routeReuseStrategy.shouldReuseRoute = () => {
+        return false;
+    };
+  }
 
   ngOnInit() {
     this.languageService.setDefaultLanguage();
     this.navigateToStartingPositionOnPage();
     this.titleAndMetaTagsService.useTitleMetasData();
     this.searchSearch.searchSubject.subscribe(this.openSearchSubscription.bind(this));
     this.uiActionsService.stopScrollingSubject.subscribe(data => this.toggle = data);
+    this.router.events.subscribe((evt) => {
```

Comment:
you add the almost the same method in ngOnInit as navigateToStartingPositionOnPage(), why?
As you can use existed method and just update it,
![image](https://user-images.githubusercontent.com/56609118/89077872-da069200-d38b-11ea-80c0-2ddb4282b4a0.png)